backtester/backtest/backtest.go

```diff
@@ -177,13 +177,12 @@ func NewFromConfig(cfg *config.Config, templatePath, output string, bot *engine.
 	if err != nil {
 		return nil, err
 	}
+	bt.Strategy.SetDefaults()
 	if cfg.StrategySettings.CustomSettings != nil {
 		err = bt.Strategy.SetCustomSettings(cfg.StrategySettings.CustomSettings)
 		if err != nil && !errors.Is(err, base.ErrCustomSettingsUnsupported) {
```

Comment:
Should the backtester continue with default if the user has put in bad settings? I think it could lead to a user assuming the results are for their custom settings when they are not.

Reply:
That's a good point, probably not. If there is a big data set to source it might be better to just return an error as not to take up resources.

Reply:
What error?